packages/grid/x-grid-modules/src/components/row.tsx

```diff
@@ -12,6 +13,8 @@ export interface RowProps {
 export const Row: React.FC<RowProps> = ({ selected, id, className, rowIndex, children }) => {
   const cssClasses = (selected ? 'selected ' : ' ') + (className || '');
```

Comment:
To match the convention of the other components:

```suggestion
  const cssClasses = clsx(ROW_CSS_CLASS, {
   'Mui-selected': selected,
  }, className);
```